cime_config/testdefs/testlist_clm.xml

```diff
@@ -3558,5 +3569,82 @@
     </options>
   </test>
 
+  <test name="SMS_Lm25" grid="f10_f10_mg37" compset="IHistClm60BgcCrop" testmods="clm/RxCropCalsNoAdapt">
+    <machines>
+      <machine name="derecho" compiler="intel" category="crop_calendars"/>
```

Comment:
Should any of the tests added to the crop_calendars testlist also be added to ctsm_sci?